config/permission.php

```diff
@@ -0,0 +1,186 @@
+<?php
```

Comment:
Did you change any of this file? If not, then we don't need to commit it. If you did, then we only need to add the missing array keys.

Reply:
Only thing that changed is the Role class path.

Reply:
Okay, so only keep that single key in this file so it overwrites it (Laravel 11).

```php
<?php

return [
    'models' => [
        'role' => \App\Models\Role::class,
    ],
];
```

So this can be the whole file.

Reply:
Already done. :)